packages/buendia-backup/data/usr/bin/buendia-restore

```diff
@@ -56,19 +61,19 @@ if [ -b "$root" ]; then
     mkdir -p "$mnt_dir"
     mount "$root" "$mnt_dir"
 
-    root="$mnt_dir/restore"
+    # Find the latest backup directory on the device, or else.
```

Comment:
How do we make sure that a backup doesn't start running during the restore and start backing up to the same directory that the restore is trying to restore from?  Do we need to rename the backup directory once we have decided to restore from it, and/or impose a lock so that backup and restore cannot occur concurrently?

Reply:
I thought of this but considered it a pretty unlikely race condition. But since you brought it up, I prevented the race by having `buendia-backup` and `buendia-restore` use the same lock file.